demo/api/src/news/generated/dto/news.sort.ts

```diff
@@ -4,7 +4,7 @@ import { SortDirection } from "@comet/cms-api";
 import { Field, InputType, registerEnumType } from "@nestjs/graphql";
 import { IsEnum } from "class-validator";
 
-export enum NewsSortField {
+enum NewsSortField {
```

Comment:
@dkarnutsch argued that the sort enum may be used in scaffolded code here: https://github.com/vivid-planet/comet-starter/pull/613. Maybe we should add a knipignore instead?

Reply:
ok, thats basically like i said -> then i will revert this change, because we do not really know if someone will be using those enums.

Reply:
sorry, my bad.

Reply:
shall we add explicit knip ignore to the enum field? or shall we globally ignore all generated/** files? 

I would prefer to ignore all generated files from linting.

Reply:
> I would prefer to ignore all generated files from linting.

Then let's do that. Please ignore the files generated by the CRUD Generators

Reply:
done. ignored generated directories in demo/api and demo/admin